index.html

```diff
@@ -2170,9 +2170,10 @@ <h2>Assertion</h2>
         <h2>Key Agreement</h2>
 
         <p>
-The `keyAgreement` <a>verification relationship</a> is used to specify how
-to encrypt information to the <a>DID subject</a>, such as for the purposes
-of establishing a secure communication channel with the recipient.
+The `keyAgreement` <a>verification relationship</a> is used to specify how an
+entity can generate encryption material in order to secretly transmit
```

Comment:
```suggestion
entity can generate encryption material in order to transmit confidential
```

Just a small suggestion (I don't think the transmission itself is necessarily secret, only its content). Anyway, also fine without this change.